src/styles/main.scss

```diff
@@ -190,6 +190,7 @@ html,
 body {
 	height: 100%;
 	overflow: hidden;
+    touch-action: pan-x pan-y;
```

Comment:
Make sure to fix indentation. I'm not sure how this wasn't caught by Prettier though, can you double check Prettier is running on save?

Reply:
Done. I have run npm, however, I don't seem to see prettier running on commits. I'll dig into this to find out what's going on.